src/Raven.Server/Documents/CountersStorage.cs

```diff
@@ -45,14 +52,22 @@ public unsafe class CountersStorage
         private enum CountersTable
         {
             // Format of this is:
-            // lower document id, record separator, lower counter name, record separator, 16 bytes dbid
-            CounterKey = 0,
-            Name = 1, // format of lazy string key is detailed in GetLowerIdSliceAndStorageKey
-            Etag = 2,
-            Value = 3,
-            ChangeVector = 4,
-            Collection = 5,
-            TransactionMarker = 6
+            // lower document id, record separator, prefix 
+            DocumentId = 0,
```

Comment:
CounterKey, instead. This isn't the document id

Reply:
done